packages/containers/psammead-timestamp-container/README.md

```diff
@@ -0,0 +1,62 @@
+
+## Description
+`psammead-timestamp-container` is a React container that offers relative and absolute times, with timezone support - using moment-timezone. Relative times are currently hard-coded to English (e.g. 3 minutes ago). `psammead-timestamp-container` returns a single time element. 
+
+## When to use this component
+`psammead-timestamp-container` is intended to be used when a single time DOM element that has the possibility of having varying time formats and timezones is required.
+
+## Installation
+`npm install @bbc/psammead-timestamp-container`
+
+## Props
+
+| Argument  | Type                | Required | Default | Example         |
+|-----------|---------------------|----------|---------|-----------------|
+| timestamp | number | Yes   | N/A | `1530947227000` |
+| dateTimeFormat | string | Yes | N/A | `YYYY-MM-DD` |
+| isRelative | boolean | No | `false` | `true` |
+| format | string | No | `null` | `D MMMM YYYY, HH:mm z` |
+| timezone | string | No | `'Europe/London'` | `'Europe/London'` |
+| prefix | string | No | `null` | `Updated` |
+| suffix | string | No | `null` | `This is a suffix` |
+| script | object | Yes | N/A | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, } |
+
+## Usage
+```jsx
+import { latin } from '@bbc/gel-foundations/scripts';
+
+const WrappingContainer = () => (
+  <Timestamp
+    timestamp={1530947227000}
+    dateTimeFormat="YYYY-MM-DD"
+    isRelative={false}
+    format="D MMMM YYYY"
+    timezone="Europe/London"
+    prefix="Updated"
+    suffix="."
+    script={latin}
+  />
+);
+```
+
+## Accessibility notes
+
+The usage of the semantic `time` element within psammead-timestamp component can result in strange behaviours when using assistive technology such as VoiceOver for iOS. Specifically, it results in a "text splitting" behaviour, where any text in the prefix is read out, but the value inside the semantic `time` element require an additional swipe before being read out.
```

Comment:
What do you mean by a prefix here? Like the sentence before or part of the timestamp?

Reply:
Here it means the sentence before